joomla/joomla-cms
# [4.3] Button text color issue

Pull Request for Issue #40306

### Summary of Changes

Changes some code in _global.scss file

### Testing Instructions

- See #40306
- See #40420
- Build: npm ci
- Clear the browser cache (historie) after the patch if necessary

### Actual result BEFORE applying this Pull Request

Text NOT continues to be read.

### Expected result AFTER applying this Pull Request

Text continues to be read.

### Link to documentations
Please select:


Thanks @joomlaphp and @dgrammatiko
